report_github_metrics.rb

```diff
@@ -28,10 +28,11 @@ def collect_workflow_metrics(workflow_run, jobs, tags)
            else
              "failure"
            end
+  is_retry = workflow_run["run_attempt"] > 1
```

Comment:
Reference: See `github.run_attempt` under https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

Reply:
Thanks for checking, I just confirmed that `run_started_at` does indeed get updated for the retry so we should be good.